src/solvers/dgsem_p4est/dg_3d_parallel.jl

```diff
@@ -0,0 +1,228 @@
+# By default, Julia/LLVM does not use fused multiply-add operations (FMAs).
```

Comment:
I haven't checked the algorithms in this file, but they are essentially what is available in 2D but with additional indices, is that correct?